### PR TITLE
fully psr2 compatible in interface/main/calendar/add_edit_event.php

### PR DESCRIPTION
**Pull request by bradymiller**
just a couple quick code changes towards full psr2

### CODE REVIEW

File: interface/main/calendar/add_edit_event.php [hunk 1]
```diff
@@ -1451,9 +1448,7 @@ function find_available(extra) {
         </td>
         <td nowrap>
             &nbsp;&nbsp;
-            <input   type='radio' name='form_allday' onclick='set_allday()' value='0' id='rballday2' <?php if ($thisduration != 1440) {
```
Comment (robertdown) [inline interface/main/calendar/add_edit_event.php]: Extraneous spaces before `type`

File: interface/main/calendar/add_edit_event.php [hunk 2]
```diff
@@ -1765,10 +1761,8 @@ function isRegularRepeat($repeat)
       repeating mechanism is being used, and load settings accordingly.
       */
         ?>
-   <input   type='checkbox' name='form_repeat' id="form_repeat" onclick='set_repeat(this)' value='1'<?php if (isRegularRepeat($repeats)) {
-        echo " checked";
-} ?>/>
-   <input   type='hidden' name='form_repeat_exdate' id='form_repeat_exdate' value='<?php echo attr($repeatexdate); ?>' /> <!-- dates excluded from the repeat -->
```
Comment (robertdown) [inline interface/main/calendar/add_edit_event.php]: Extraneous whitespace

File: interface/main/calendar/add_edit_event.php [hunk 3]
```diff
@@ -1816,9 +1810,7 @@ function isRegularRepeat($repeat)
 <tr id="days_every_week_row">
     <td></td>
     <td></td>
-    <td><input    type='checkbox' id='days_every_week' name='days_every_week' onclick='set_days_every_week()' <?php if (isDaysEveryWeek($repeats)) {
```
Comment (robertdown) [inline interface/main/calendar/add_edit_event.php]: Extraneous whitespace

File: interface/main/calendar/add_edit_event.php (hunk 1 above)
Comment (bradymiller) [inline interface/main/calendar/add_edit_event.php]: these were removed as part of the PR :)
(same for the other places)

**Comment by robertdown** [comment]
...you're right. I've clearly forgotten how to do a code review

**Pull request merged**